### ISSUE TITLE
Issue with dropping a table that contains upper case letters in the name.

### ISSUE THREAD

**Issue by JonoPrest** [drizzle-team/drizzle-kit-mirror#43]
These are the steps to recreate the issues:

I create a table called "myTest1" (contains uppercase "T").
The following migration is generated by drizzle-kit in a file called "0000_zippy_rage.sql":

```sql
CREATE TABLE IF NOT EXISTS "myTest1" (
	"id" text PRIMARY KEY NOT NULL,
	"owner" text,
	"displayName" text,
	"imageUrl" text,
	"updatesCount" text
);

```

Running migrate on this works perfectly.

Next I change the name in my schema to "myTest2" and select create table instead of rename table: 
<img width="533" alt="Screenshot 2023-05-02 at 13 16 47" src="https://user-images.githubusercontent.com/65739024/235653401-752d6efc-c7e9-4815-929e-a8d6a4c96766.png">

The following migration is generated in a file called "0001_wandering_tana_nile.sql":
```sql
CREATE TABLE IF NOT EXISTS "myTest2" (
	"id" text PRIMARY KEY NOT NULL,
	"owner" text,
	"displayName" text,
	"imageUrl" text,
	"updatesCount" text
);

DROP TABLE myTest1;
```

Note how the command `DROP TABLE myTest1;` does not use quotes around "myTest1".

When I run this migration I get this error in my terminal: `error: table "mytest1" does not exist`

As far as I know without quotes this command won't work. Perhaps it's a simple matter of adding these quotes into drizzlekit functionality.

**Comment by AndriiSherman** [comment]
What version of drizzle kit are you using?

**Comment by AndriiSherman** [comment]
Fixed in `drizzle-kit@0.18.1`
